src/helpers/utils.ts

```diff
@@ -45,8 +76,18 @@ export async function getEmailAddresses(email: string) {
   ]);
 }
 
-export async function getUniqueEmails() {
-  return await db.queryAsync('SELECT DISTINCT email FROM subscribers WHERE verified > 0');
+export async function getUniqueEmails(subscriptionType: string) {
+  return await db.queryAsync(
+    `SELECT DISTINCT email FROM subscribers WHERE verified > 0 AND JSON_CONTAINS(subscriptions, '${JSON.stringify(
+      sanitizeSubscriptions(subscriptionType)[0]
+    )}')`
+  );
+}
```

Comment:
Aren't you missing the `JSON.stringify` ?

Reply:
it's redundant for this case, but we can add it. Since by stringifying string, you receive the string with quotes: 
```js
const str = 'test';
console.log(JSON.stringify(str) === '"test"'); // true
```
if you think that it should be added, then:
```suggestion
export async function getUniqueEmails(subscriptionType: string) {
  const subscription = sanitizeSubscriptions(subscriptionType)[0];
  const stringifiedSubscription = JSON.stringify(subscription);
  
  return await db.queryAsync(
    `SELECT DISTINCT email FROM subscribers WHERE verified > 0 AND JSON_CONTAINS(subscriptions, '${stringifiedSubscription}')`
  );
}
```

Reply:
yes, we do need both the single and double here